OpenAI_API/Chat/Conversation.cs

```diff
@@ -122,5 +122,52 @@ public async Task<string> GetResponseFromChatbot()
 			}
 			return null;
 		}
+
+		/// <summary>
+		/// Calls the API to get a response, which is appended to the current chat's <see cref="Messages"/> as an <see cref="ChatMessageRole.Assistant"/> <see cref="ChatMessage"/>, and streams the results to the <paramref name="resultHandler"/> as they come in. <br/>
+		/// If you are on the latest C# supporting async enumerables, you may prefer the cleaner syntax of <see cref="StreamResponseEnumerableFromChatbot"/> instead.
+		///  </summary>
+		/// <param name="resultHandler">An action to be called as each new result arrives.</param>
+		public async Task StreamResponseFromChatbot(Action<string> resultHandler)
```

Comment:
Please use suffix `Async`

Reply:
`Conversation`'s `async Task<string> GetResponseFromChatbot()` doesn't have `Async` suffix. But some async methods in other classes have. So I am a little bit confused.

Reply:
The majority of the codebase uses Async suffix except for the tests, but you're right aswell. One of the questions i still had when looking through the code :) 
Per Microsoft Async suffix should not be used unless you have synchronous AND asynchronous methods for the same functionality. But to not exclude synchronous or create ambiguity it's best practice to suffix it with `Async` to always have the option open. But yeah it should be consistent heheh. #71 will help with this i guess

Reply:
Sorry, I guess I missed one

Reply:
I added "Async" suffix

Reply:
`Conversation` class already has a `GetResponseFromChatbot()` method. So I name the stream methods `StreamResponseFromChatbot` and `StreamResponseEnumerableFromChatbot`. 

With `Async` suffix it will be like `StreamResponseEnumerableFromChatbotAsync`, which seems too long.

In `CompletionEndpoint` we have `CreateCompletionAsync()` and `StreamCompletionAsync()`. So maybe this is better?

- `GetResponseAsync()`
- `StreamResponseAsync()`
- `StreamResponseEnumerableAsync()`

Can I modify the name of `GetResponseFromChatbot()` in this pull request?